src/common/legacy/src/graph_transformer.cpp

```diff
@@ -231,6 +231,9 @@ const std::map<std::string, bool> ConstTransformer::getConstLayers(const std::ve
     std::map<std::string, bool> mapConstLayers;
     // collect all const layers, which inputs are const layers.
     for (const auto& layer : sortedLayers) {
+        if (layer->insData.size() == 0) {
+            continue;
+        }
```

Comment:
I believe this condition should get all the Constant nodes that are used as inputs as they have no input data themselves.
@Maxim-Doronin what do you think about this approach?